exercises/random/challenge_058.py

```diff
@@ -0,0 +1,51 @@
+### RANDOM CHALLENGE ###
+
+"""
+058
+Make a maths quiz that asks five questions by randomly generating two whole number 
+to make the question (e.g. [num1] + [num2]). Ask the user to enter the answer.
+If they get it right add a point to the score. At the end of the quiz, 
+tell them how many they got correct out of five.  
+"""
+import common_functions
+
+import random
+import operator
+
+# using dictionary to take random operators
+
+
+def randomCalc(current_index, total_questions):
+    ops = {'+': operator.add,
```

Comment:
Again the short names for everything here is harder to understand than `operations` and `chosen_operator`